server/src/main/java/org/opensearch/cluster/ExtensionRequest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.cluster;
```

Comment:
Move ExtensionsRequest.java under the `extensions` directory instead of `cluster` since this request will be used for all extension points

Reply:
Good call out.